build.sbt

```diff
@@ -85,6 +85,7 @@ lazy val core = libraryProject("core")
       scalaReflect(scalaVersion.value) % Provided,
       vault,
     ),
+    unusedCompileDependenciesFilter -= moduleFilter("org.scala-lang", "scala-reflect"),
```

Comment:
This was only an issue on Scala 2.12. It very certainly is used! I don't know what's going on here.